storage/replica.go

```diff
@@ -202,7 +203,7 @@ type Replica struct {
 	// Last index applied to the state machine. Updated atomically.
 	appliedIndex uint64
 	configHashes map[int][]byte // Config map sha256 hashes @ last gossip
-	systemDBHash []byte         // sha256 hash of the system config @ last gossip
+	systemDBHash []byte         // sha1 hash of the system config @ last gossip
```

Comment:
maybe, I'd have to look into how they perform and how the usage differs.
from the existing benchmarks, adler32 might be the best bet.